hcloud/resource_hcloud_server.go

```diff
@@ -82,7 +82,7 @@ func resourceServer() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
-			"ipv6_address": {
+			"ipv6_network": {
```

Comment:
Isn’t this a breaking change?

Reply:
Yeah this is a breaking change, Nico and I discussed this and we decided to introduce this breaking change since the current behavior is not really good and usable.